### DISCUSSION TITLE
Is autoPlay false consuming bandwidth anyway ?

### DISCUSSION THREAD

**Discussion by Tom3652** [jhomlala/betterplayer#495] [Q&A]
Hello !

I simply would like to know if initializing a ```BetterPlayer``` with the ```autoPlay``` attribute set to false is buffering the video and / or consuming bandwidth anyway ( because it displays the first frame) ?  

Thanks !

**Comment by jhomlala** [comment]
Yes, it will consume bandwidth even if autoPlay is false. It won't be much but it will consume bandwidth to select data which is required to setup video.
